config/v1/types.go

```diff
@@ -220,18 +233,22 @@ type EtcdStorageConfig struct {
 
 // GenericAPIServerConfig is an inline-able struct for aggregated apiservers that need to store data in etcd
 type GenericAPIServerConfig struct {
-	// ServingInfo describes how to start serving
+	// servingInfo describes how to start serving
 	ServingInfo HTTPServingInfo `json:"servingInfo"`
 
-	// CORSAllowedOrigins
+	// corsAllowedOrigins
 	CORSAllowedOrigins []string `json:"corsAllowedOrigins"`
 
-	// AuditConfig describes how to configure audit information
+	// auditConfig describes how to configure audit information
 	AuditConfig AuditConfig `json:"auditConfig"`
```

Comment:
why did we expose those Config and info postfixes? How ugly 😖

Reply:
> why did we expose those Config and info postfixes? How ugly

shrugs.  Fixable if you want to sort it out this week.